README.md

```diff
@@ -60,7 +60,7 @@ instance as `received_logger`.
 ```ruby
 require 'optional_logger'
 
-logger = OptionalLogger.new(received_logger)
+logger = OptionalLogger::Logger.new(received_logger)
```

Comment:
Nope, can't buy into either of those as they both seem like entities so it sounds like it is a logger of options not an optional logger.